src/features/virtualized-table/DraggableVirtualizedTable.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/variables';
+
+.bdl-DraggableVirtualizedTable {
+    .ReactVirtualized__Table__row {
+        $base-row-shadow: 0 -1px 1px $bdl-gray-10;
+
+        background-color: $white;
+        // Allows row to have both top and bottom borders when dragging
+        box-shadow: $base-row-shadow;
+
+        &.is-dragging {
+            box-shadow: $base-row-shadow, 0 1px 3px $bdl-gray-20;
+        }
+    }
+
+    .ReactVirtualized__Table__rowColumn,
+    .ReactVirtualized__Table__headerColumn {
+        &.bdl-DraggableVirtualizedTable-dragHandleColumn {
+            margin-right: 0;
+        }
+    }
+}
+
+.bdl-DraggableVirtualizedTable-dragHandleColumn {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: 24px;
```

Comment:
if this is 24px for the same reason `ICON_SIZE` is 24px, then we could create a token in our variables for such a thing. there is no effort to do this right now, but something to think about